deployment/grid/terraform/control_plane/dynamodb.tf

```diff
@@ -2,102 +2,137 @@
 # SPDX-License-Identifier: Apache-2.0
 # Licensed under the Apache License, Version 2.0 https://aws.amazon.com/apache-2-0/
 
-resource "aws_dynamodb_table" "htc_tasks_state_table" {
-  name           = var.ddb_state_table
-  read_capacity  = var.dynamodb_table_read_capacity
-  write_capacity = var.dynamodb_table_write_capacity
-
-  hash_key       = "task_id"
-
-
-  attribute {
-    name = "session_id"
-    type = "S"
+module "dynamodb_table" {
+  source = "terraform-aws-modules/dynamodb-table/aws"
```

Comment:
Would it be possible to enforce the version of the module ?

Reply:
Done, will update the pull request shortly